uPortal-webapp/src/main/resources/properties/db/tables.xml

```diff
@@ -52,6 +52,10 @@
         <name>UPU_DFLT_ID_IDX</name>
         <column-ref>USER_DFLT_USR_ID</column-ref>
     </index>
+    <index>
+        <name>UPU_USER_NAME_IDX</name>
+        <column-ref>USER_NAME</column-ref>
+    </index>
```

Comment:
This one adds a `UPU_USER_NAME_IDX` index on `up_user`...

![up_user-after](https://user-images.githubusercontent.com/1127174/35597848-8a5c57ae-05dd-11e8-8359-2b0187f8b417.png)